scrape/scrape.go

```diff
@@ -267,6 +269,7 @@ func (sp *scrapePool) reload(cfg *config.ScrapeConfig) error {
 		bodySizeLimit = int64(sp.config.BodySizeLimit)
 		sampleLimit   = int(sp.config.SampleLimit)
 		bucketLimit   = int(sp.config.NativeHistogramBucketLimit)
+		maxSchema     = pickSchema(sp.config.NativeHistogramMinBucketFactor)
```

Comment:
In order to change config to `NativeHistogramMinBucketFactor`, I convert the `NativeHistogramMinBucketFactor` into `maxSchema` here, with `pickSchema` function copied from client_golang, are you ok with this implementation? 
@beorn7

Reply:
Sounds good. (I have only looked at your comment, not at the code.)